docs/changelog_for_devs.md

```diff
@@ -1,3 +1,9 @@
+# v0.3.2
+
+- Changed the all parameters of type `BalanceOf`, `MarketId` and `PoolId` in
```

Comment:
Is it required to mention the one `#[pallet::compact] categories: u16` parameter of the prediction markets function `create_categorical_market`? (look review comment below)

Reply:
Yeah, we should mention `u16`